pallets/stableswap/src/lib.rs

```diff
@@ -416,9 +423,18 @@ pub mod pallet {
 			Pools::<T>::try_mutate(pool_id, |maybe_pool| -> DispatchResult {
 				let mut pool = maybe_pool.as_mut().ok_or(Error::<T>::PoolNotFound)?;
 
+				ensure!(
+					pool.final_block <= current_block,
+					Error::<T>::AmplificationChangeNotCompleted
+				);
+				ensure!(
+					pool.final_amplification.get() != final_amplification,
+					Error::<T>::SameAmplification
```

Comment:
what if you want to speed up or slow down ramp up? this check seems not very effective as it could be bypassed by just changing final amp by 1

Reply:
this just ensures that you dont set the same value as there is now as it does not make any change

Reply:
it does, as you can set different end block therefore slow down or hasten the change

Reply:
yes. now it is different - this was a case when you had to wait for previous change to finish. 

now it makes sense to ensure that the new final amplification != initial ampfliciation, isnt it ?